tests/pipeline/resources/defaults.yaml

```diff
@@ -104,8 +104,7 @@ should-inflate:
           retention.ms: "-1"
 
 kafka-connector:
-  name: "sink-connector"
-  namespace: "example-namespace"
+  name: sink-connector
```

Comment:
Change the `kubernetes-app` in this defaults.yaml to `helm-app`

Reply:
why?

Reply:
I renamed kafka-connector → kafka-sink-connector though

Reply:
but it doesn't contain anything Helm-specific in this case, only `name` and `namespace`. I moved `name` to `pipeline-component` defaults now, but `namespace` is fine for all `kubernetes-app` imo

Reply:
Okay makes sense, I thought that the kubernetes-app is a more abstract type that is better to replace with a concrete implementation (like helm-app).